backend/src/main/java/com/karankumar/bookproject/backend/model/Tag.java

```diff
@@ -50,7 +51,7 @@
     @Column(unique = true)
     private String name;
 
-    @ManyToMany(fetch = FetchType.EAGER, mappedBy = "tags")
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.REMOVE, mappedBy = "tags")
```

Comment:
Could we remove the new cascade type? We don't want to remove a book if a tag is deleted

Reply:
Removed but this introduces a set of new test errors (which was the orif=ginal reason I added the CasacdeType.REMOVE). The test errors (not failures) can be seen here --> https://pastebin.com/yC60U37C.